module/x/gravity/keeper/test_common.go

```diff
@@ -180,23 +180,23 @@ var (
 	}
 
 	// TestingGravityParams is a set of gravity params for testing
-	TestingGravityParams = types.Params{
-		GravityId:                                 "testgravityid",
-		ContractSourceHash:                        "62328f7bc12efb28f86111d08c29b39285680a906ea0e524e0209d6f6657b713",
-		BridgeEthereumAddress:                     "0x8858eeb3dfffa017d4bce9801d340d36cf895ccf",
-		BridgeChainId:                             11,
-		SignedBatchesWindow:                       10,
-		SignedSignerSetTxsWindow:                  10,
-		UnbondSlashingSignerSetTxsWindow:          15,
-		EthereumSignaturesWindow:                  10,
-		TargetEthTxTimeout:                        60001,
-		AverageBlockTime:                          5000,
-		AverageEthereumBlockTime:                  15000,
-		SlashFractionSignerSetTx:                  sdk.NewDecWithPrec(1, 2),
-		SlashFractionBatch:                        sdk.NewDecWithPrec(1, 2),
-		SlashFractionEthereumSignature:            sdk.NewDecWithPrec(1, 2),
-		SlashFractionConflictingEthereumSignature: sdk.NewDecWithPrec(1, 2),
+	ethereumChainParams = types.ChainParams{
+		GravityId:                            "testgravityid",
+		ContractSourceHash:                   "62328f7bc12efb28f86111d08c29b39285680a906ea0e524e0209d6f6657b713",
```

Comment:
Happy to do that if you don't think it's needed

Reply:
Those three elements are pretty related, and since the contract is deployed after the chain is live, they'd need to be set via governance. We haven't set them so far, and since the chain can't interact with the bridge directly, they really only serve as a store of information for external parties like the orchestrator, who currently set the contract address and other values in their configs.

Reply:
Removed